tmail-backend/jmap/extensions-rabbitmq/src/main/java/com/linagora/tmail/james/jmap/RabbitMQEmailAddressContactSubscriber.java

```diff
@@ -32,7 +34,8 @@
 import reactor.rabbitmq.Receiver;
 import reactor.rabbitmq.Sender;
 
-public class RabbitMQEmailAddressContactSubscriber implements Closeable {
+public class RabbitMQEmailAddressContactSubscriber implements Startable, Closeable {
+    public static final String RABBITMQ_CONTACT_CONFIGURATION_NAME = "rabbitmq-contact";
```

Comment:
We will have one more file for config contact sync. 
` rabbitmq-contact.properties"

Reply:
Ref: https://github.com/linagora/tmail-backend/pull/403#issuecomment-1106063906

Reply:
Reuse `rabbitmq.properties`